composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "doctrine/key-value-store",
+    "name": "njuko/key-value-store",
```

Comment:
Not supposed to be merge with master. Sorry

-- 
David Ribes

De: Emanuele Minotto <notifications@github.com> <notifications@github.com>
Répondre: doctrine/KeyValueStore
<reply@reply.github.com>
<reply@reply.github.com>
Date: 30 juin 2016 at 11:31:47
À: doctrine/KeyValueStore <keyvaluestore@noreply.github.com>
<keyvaluestore@noreply.github.com>
Cc: Ribes <david.ribes@njuko.com> <david.ribes@njuko.com>, Author
<author@noreply.github.com> <author@noreply.github.com>
Sujet:  Re: [doctrine/KeyValueStore] Fix table creation (#81)

In composer.json
<https://github.com/doctrine/KeyValueStore/pull/81#discussion_r69101618>:

> @@ -1,5 +1,5 @@
>  {
> -    "name": "doctrine/key-value-store",
> +    "name": "njuko/key-value-store",

???

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/doctrine/KeyValueStore/pull/81/files/8352ff4fbb81bd449ee786dbf30af6700ee9b32e#r69101618>,
or mute the thread
<https://github.com/notifications/unsubscribe/ACtHA9PRg5X3bTG9BpkEOZhPUTRQdAJkks5qQ4z8gaJpZM4JB9AG>
.